### PR TITLE
*: Get ddl owner ID

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -160,10 +160,13 @@ type Executor interface {
...
 	baseExecutor
 
-	schema  *expression.Schema
-	ddlInfo *inspectkv.DDLInfo
-	bgInfo  *inspectkv.DDLInfo
-	done    bool
+	ddlOwnerID string
+	bgOwnerID  string
+	selfID     string
+	schema     *expression.Schema
```
Comment (hanfei1991) [inline executor/executor.go]: schema has existed in `baseExecutor`

Comment (zimulala) [inline executor/executor.go]: It's the old logic, I don't change it.

Comment (hanfei1991) [inline executor/executor.go]: You can remove it by chance.

**Comment by zimulala** [comment]
PTAL @hanfei1991 @XuHuaiyu @tiancaiamao

**Comment by hanfei1991** [comment]
LGTM

File: executor/executor.go [hunk 2]
```diff
@@ -172,24 +174,24 @@ func (e *ShowDDLExec) Next() (*Row, error) {
...
-	var bgOwner, bgJob string
+	var bgJob string
 	if e.bgInfo.Job != nil {
 		bgJob = e.bgInfo.Job.String()
 	}
 
 	row := &Row{}
 	row.Data = types.MakeDatums(
 		e.ddlInfo.SchemaVer,
-		ddlOwner,
```
Comment (shenli) [inline executor/executor.go]: What's different between ddlOwner and e.ddlOwnerID?

Comment (zimulala) [inline executor/executor.go]: The original `ddlOwner` contains `ddlOwnerID` and `LastUpdateTS` fields.

**Comment by shenli** [comment]
LGTM

**Pull request merged**